reactivesocket-client/src/main/java/io/reactivesocket/client/ClientBuilder.java

```diff
@@ -104,16 +104,17 @@ public ClientBuilder withConnectTimeout(long timeout, TimeUnit unit) {
     }
 
     public ClientBuilder withBackupRequest(double quantile) {
-        return new ClientBuilder(
-            name,
-            executor,
-            requestTimeout, requestTimeoutUnit,
-            connectTimeout, connectTimeoutUnit,
-            quantile,
-            retries, retryThisException,
-            connector,
-            source
-        );
+        throw new IllegalArgumentException("BackupRequest is experimental/disabled.");
```

Comment:
Do you want to remove these methods for now?

We are anyways breaking API backward compatibility with the RxNetty chane.